userguides/WritingHiveQLUnitTests.md

```diff
@@ -79,7 +79,7 @@ The TestDataLoader provides utility methods accesible from within test methods f
     public void test() {
         loader.loadDataIntoTable("src", new LocalFileResource("C:/cygwin64/home/K00001/testdata.txt"), "");
 
-        HiveContext sqlContext = hiveServer.getHiveContext();
+        HiveContext sqlContext = hiveServer.getSqlContext();
         Row[] results = sqlContext.sql("SELECT key FROM src WHERE key = 5").collect();
```

Comment:
Since we are changing the *ExpressionScript runScriptReturnResults return type from Row[] to List<Row>, it would be best to update the example use collectAsList as well.